### PR TITLE
Added different FoV lenses to the arducam cameras page

### PR DESCRIPTION
**Pull request by Erol444**
@alex-luxonis  I added you just so you can double-check that everything is compatible as I wrote it:) Thanks!

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Thanks!  Looks great at first glance!

File: docs/source/pages/arducam.rst [hunk 1]
```diff
@@ -9,16 +9,29 @@ Overview
 ********
 
 `ArduCam <https://www.arducam.com/product-category/opencv-ai-kit/>`__ has built a series of M12-capable and C/CS-capable camera boards that work with DepthAI
-and megaAI. A variety of M12-mount options allow to match lenses in many projects. More about how to get different field of view `here <https://docs.luxonis.com/en/latest/pages/faq/#how-do-i-get-different-field-of-view-or-lenses-for-depthai-and-megaai>`__.
+and megaAI. A variety of M12-mount options allow to match lenses in many projects.
 
-Models
-******
+OAK-FCC compatible cameras
```
Comment (alex-luxonis) [inline docs/source/pages/arducam.rst:14]: OAK-**FCC** -> **FFC**? Also for the `:ref:` above

File: docs/source/pages/arducam.rst [hunk 2]
```diff
@@ -9,16 +9,29 @@ Overview
...
+**************************
+
+- `M12-Mount Global Shutter Grayscale OV9282 <https://www.arducam.com/product/arducam-1mp-ov9282-global-shutter-mono-mipi-camera-module-22pin-for-depthai-oak-dm1090ffc/>`__
+- `M12-Mount Global Shutter Color OV9782 <https://www.arducam.com/product/arducam-1mp-ov9782-global-shutter-color-mipi-camera-module-22pin-for-depthai-oak-dm1090ffc/>`__
+
+These two cameras are compatible with the :ref:`OAK-FFC-3P <dm1090ffc>`. And since they have **M12 mount**,
+you can use a variety of **differnet lenses** (for **custom FoV**), for example:
```
Comment (alex-luxonis) [inline docs/source/pages/arducam.rst:21]: Typo: differ**ne**t

File: docs/source/pages/DM1090.rst [hunk 3]
```diff
@@ -14,6 +14,9 @@ The DM1090FFC baseboard has three FFC interfaces which allow for two 2-lane MIPI
 Please note that only revision R1M1E1 and newer camera modules can be connected to the DM1090FFC. With DM1090FFC we gained support for 22-pin RPi camera 
 interface. For that you will need a FFC from Arducam, which converts 26-pin Luxonis camera pinout to 22 pin RPi camera pinout.
```
Comment (cafemoloko) [inline docs/source/pages/DM1090.rst:15]: it would be nice to add a link to ArduCam FFC

File: docs/source/pages/arducam.rst [hunk 4]
```diff
@@ -9,16 +9,29 @@ Overview
 ********
 
 `ArduCam <https://www.arducam.com/product-category/opencv-ai-kit/>`__ has built a series of M12-capable and C/CS-capable camera boards that work with DepthAI
-and megaAI. A variety of M12-mount options allow to match lenses in many projects. More about how to get different field of view `here <https://docs.luxonis.com/en/latest/pages/faq/#how-do-i-get-different-field-of-view-or-lenses-for-depthai-and-megaai>`__.
```
Comment (cafemoloko) [inline docs/source/pages/arducam.rst:12]: Shall we update product names here?

Comment (Luxonis-Brandon) [reply docs/source/pages/arducam.rst:12]: Yes please do.  We need to scrub the FAQ/etc. as well.

File: docs/source/pages/DM1090.rst (hunk 3 above)
Comment (Erol444) [reply docs/source/pages/DM1090.rst:15]: https://shop.luxonis.com/collections/all/products/rpi-hq-camera-imx477-adapter-kit <- Is this the one?

**Review by cafemoloko — APPROVED** [review]

File: docs/source/pages/arducam.rst (hunk 1 above)
Comment (alex-luxonis) [reply docs/source/pages/arducam.rst]: Sorry, I meant the F**C**C typo here. Okay to have OAK-FFC

**Pull request merged**